issuesync.rb

```diff
@@ -226,6 +229,14 @@ def updated_at() Time.parse(data['updated_at']) end
     def has_comments?() !data['comments'].zero? end
     def open?() data['state'] == 'open' end
     def closed?() !open? end
+
+    def labels()
```

Comment:
The method definitions above use parens because I tried to cram them on one line, and in that case parens are required as a delimiter.